pom.xml

```diff
@@ -34,6 +34,7 @@
         <commons.version>3.6</commons.version>
         <commons.collections.version>4.1</commons.collections.version>
         <commons.compress.version>1.18</commons.compress.version>
+        <httpclient.version>4.5.5</httpclient.version>
```

Comment:
Do we actually need to add this as a new dependency? I see that we have it already in the project, used in `SharethroughUriBuilderUtil#21` without declaring explicit dependency (probably added as a part of some other dependency).

Reply:
Yes, @schernysh asked me to show usage of it explicitly.